SocialCareCaseViewerApi.Tests/V1/Gateways/Database/CreatePersonalRelationshipTests.cs

```diff
@@ -136,5 +136,37 @@ public void CreatesAPersonalRelationshipDetails()
             details?.PersonalRelationshipId.Should().Be(response.Id);
             details?.Details.Should().Be(request.Details);
         }
+
+        [Test]
+        public void AuditsThePersonalRelationship()
+        {
+            var (person, otherPerson) = PersonalRelationshipsHelper.SavePersonAndOtherPersonToDatabase(DatabaseContext);
+            var type = DatabaseContext.PersonalRelationshipTypes.FirstOrDefault(prt => prt.Description == "parent");
+            var request = PersonalRelationshipsHelper.CreatePersonalRelationshipRequest(
+                person.Id, otherPerson.Id, type.Id, type.Description
+            );
+
+            var response = _databaseGateway.CreatePersonalRelationship(request);
+
+            var personalRelationship = DatabaseContext.PersonalRelationships.FirstOrDefault();
+            personalRelationship?.CreatedAt.Should().NotBeNull();
```

Comment:
I'd rather our test not just confirm the value isn't null.
However this is a bigger piece of work so I'll create a ticket.

I expect the issue is that CreatedAt is happening at runtime during the actual object creation so we have no way of validating the actual value against something we expected. The datetime will be new every time the test runs. 
We need something to abstract DateTime which we can then mock for our tests.

https://stackoverflow.com/questions/2425721/unit-testing-datetime-now